elyra/tests/pipeline/resources/sample_pipelines/pipeline_valid_with_pipeline_default.json

```diff
@@ -60,6 +60,22 @@
               "description": "Notebook file"
             }
           }
+        },
+        {
+          "id": "{{uuid-DeriveFromTestOperator2}}",
+          "type": "execution_node",
+          "op": "local-file-catalog:8371f5970c7b:DeriveFromTestOperator",
+          "app_data": {
+            "label": "{{label}}",
+            "component_parameters": {
+              "mounted_volumes": []
+            },
+            "component_source": "{{component_source}}",
+            "ui_data": {
+              "label": "{{label}}",
+              "description": "Notebook file"
```

Comment:
It doesn't impact the test outcome but the value should not be `Notebook file` since this node uses a custom component. (Same in line 60)

Reply:
good catch!